src/sage/schemes/elliptic_curves/hom.py

```diff
@@ -722,6 +724,108 @@ def as_morphism(self):
         Y_affine = Curve(self.codomain()).affine_patch(2)
         return X_affine.hom(self.rational_maps(), Y_affine).homogenize(2)
 
+    def matrix_on_subgroup(self, domain_gens, codomain_gens=None):
+        r"""
+        Return the matrix as which this isogeny acts on the
+        `n`-torsion subgroup with respect to the given bases.
+
+        (This is useful for tasks such as computing a set of
```

Comment:
I would leave out the parenthetical remark.  Other people might have other uses!

Reply:
Basically this is a postive review, I just recommend one change of wording and make another minor suggestion, both in docstrings.

Reply:
Thanks for your comments! I converted the parenthetical remark to an example (making use of functionality recently merged in #34968), which is more helpful anyway.

Reply:
Perfect!